lib/active_admin.rb

```diff
@@ -1,4 +1,5 @@
-require 'active_support/core_ext'
+require 'active_support'
+require 'active_model'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.